src/features/posts/postSlice.js

```diff
@@ -207,6 +260,17 @@ const postSlice = createSlice({
     [unLikedUserPost.pending]: (state, action) => {
       state.unlikePostStatus = "pending";
     },
+    [addComments.fulfilled]: (state, action) => {
+      state.comments = state.comments.concat(action.payload);
+    },
+    [getAllComments.fulfilled]: (state, action) => {
+      state.comments = action.payload;
+    },
+    [deleteComments.fulfilled]: (state, action) => {
+      state.comments = state.comments.filter(
+        (comment) => comment.id !== action.payload
+      );
```

Comment:
Got a new logic of deleting comments!! and API responses created through async thunk also cleared the query